src/index.ts

```diff
@@ -47,7 +54,7 @@ const run = async () => {
 			return;
 		}
 
-		if (action !== 'opened') {
+		if (!forceUpdate && action !== 'opened') {
 			console.log('Skipping action to ensure we only comment once.');
 			return;
 		}
```

Comment:
Although I'm not sure why this code and option is here because it's equivalent to using this in the github action config:

```yaml
on:
  pull_request:
    types:
      - opened
```

Reply:
Doing it as above is preferable since then you save a minute of billable time running the action only for it to do nothing

Reply:
It's mostly to make it so if people don't configure it properly it still won't spam your PRs.